src/Resources/contao/classes/Backend.php

```diff
@@ -714,9 +714,13 @@ public static function findSearchablePages($pid=0, $domain='', $blnIsSitemap=fal
 	 * @param string  $strUuid
 	 * @param string  $strPtable
 	 * @param integer $intPid
+	 *
+	 * @deprecated Deprecated since Contao 4.4, to be removed in Contao 5.0.
 	 */
 	public static function addFileMetaInformationToRequest($strUuid, $strPtable, $intPid)
 	{
+		@trigger_error('Using Backend::addFileMetaInformationToRequest() has been deprecated and will no longer work in Contao 5.0.', E_USER_DEPRECATED);
```

Comment:
Nothing. The elements now directly use the meta data, so we do not need this workaround anymore.